openml/extensions/sklearn/extension.py

```diff
@@ -206,7 +206,9 @@ def remove_all_in_parentheses(string: str) -> str:
     ################################################################################################
     # Methods for flow serialization and de-serialization
 
-    def flow_to_model(self, flow: 'OpenMLFlow', initialize_with_defaults: bool = False) -> Any:
+    def flow_to_model(self, flow: 'OpenMLFlow',
+                      initialize_with_defaults: bool = False,
+                      strict_version: Optional[bool] = True) -> Any:
```

Comment:
Why is this optional here, while it is not for `get_flow`? Also, could you please add this argument to the base class, too?